MapboxNavigationTests/InstructionPresenterTests.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import XCTest
+@testable import MapboxNavigation
+import MapboxDirections
```

Comment:
You can do testable import MapboxDirections in order to use the private strongly typed init methods instead of json. (As long as you use `--configuration Debug` when you do `carthage bootstrap/update`.